storefront/generateRouteRewrites.js

```diff
@@ -0,0 +1,17 @@
+const config = require('./theme.json');
+
+const generateRouteRewrites = () => {
```

Comment:
Actually no real rewrites are generated because we don't have a CMS and the slugs for assortments and products already come localized, this is just an indirection to next.config.js here